### PR TITLE
fix QR login error - PN-12155

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
fix error toast when go on notification with aar login from QR
## List of changes proposed in this pull request
- insert in rejectWithValue null

## How to test
https://cittadini.uat.notifichedigitali.it/auth/login?aar=VEFXUC1SSFpELUVIVEUtMjAyNDA3LUctMV9QRi1iMzJlNDkyMC02ZmYzLTQ4NzItODAxOC1kNjBhNGU1ODI3ZjlfYTY2Yjk0NWEtMDM5OS00Mzg1LWE0N2EtZjU3MzhlNjg2MzVm

insert wrong credentials when login

### CODE REVIEW

File: packages/pn-personafisica-webapp/src/redux/notification/actions.ts [hunk 1]
```diff
@@ -275,7 +275,7 @@ export const exchangeNotificationQrCode = createAsyncThunk<BffCheckAarResponse,
       const response = await notificationReceivedApiFactory.checkAarQrCodeV1(params);
       return response.data;
     } catch (e: any) {
-      return rejectWithValue(parseError(e));
```
Comment (mflauti) [inline packages/pn-personafisica-webapp/src/redux/notification/actions.ts:278]: Restore the old version and use the AppResponsePublisher to intercept the error and manage it properly. Do the same for PG.

**Review by mflauti — CHANGES_REQUESTED** [review]

File: packages/pn-personafisica-webapp/src/redux/notification/actions.ts (hunk 1 above)
Comment (AndreaCimini90) [reply packages/pn-personafisica-webapp/src/redux/notification/actions.ts:277]: AppResponsePublisher can be useful when we have to manage the error in a way that is different from the standard way (i.e. show the toast error).
In this case, it's ok to show the error but we have to show it with a different message. So ask to Daniela or Norma the copy (tag them in the issue) and after manage the error through the AppError

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personafisica-webapp/src/navigation/AARGuard.tsx [hunk 2]
```diff
@@ -52,6 +52,14 @@ const AARGuard = () => {
     }
   }, [aar]);
 
+  useEffect(() => {
+    AppResponsePublisher.error.subscribe('exchangeNotificationQrCode', ()=>false);
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/navigation/AARGuard.tsx:56]: the code is ok. we have to manage only the case of 404 and PN_DELIVERY_NOTIFICATIONNOTFOUND. In the function ()=>false, you should have the error object and you can use it to check if the error is the error with code PN_DELIVERY_NOTIFICATIONNOTFOUND

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personafisica-webapp/src/navigation/AARGuard.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-personafisica-webapp/src/navigation/AARGuard.tsx:56]: doine

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**